README.md

```diff
@@ -14,7 +14,11 @@ https://github.com/js-reporters/js-reporters/issues/
 
 ## Background
 
-We on the [QUnit](http://qunitjs.com/) team have been [discussing](https://github.com/jquery/qunit/issues/531) the possibility of working with other JS test frameworks, especially those that can be run client-side (e.g. Mocha, Jasmine, Intern, Buster, etc.), to agree upon a "Common Reporter Interface" so that we could hopefully share Reporter plugins between testing frameworks. This would also benefit high-level consumers of the frameworks such as Karma, BrowserStack, SauceLabs, Testling, etc.
+We on the [QUnit](http://qunitjs.com/) team have been [discussing](https://github.com/jquery/qunit/issues/531) the
+possibility of working with other JS test frameworks, especially those that can be run client-side (e.g. Mocha, Jasmine,
+Intern, Buster, etc.), to agree upon a "Common Reporter Interface" so that we could hopefully share Reporter plugins
+between testing frameworks. This would also benefit high-level consumers of the frameworks such as Karma, BrowserStack,
+SauceLabs, Testling, etc.
```

Comment:
Why did you change everything to hard line breaks? We usually don't use those in markdown, makes it much easier to edit prose.